### DISCUSSION TITLE
Is this safe for the server?

### DISCUSSION THREAD

**Discussion by DanKawazko** [victornpb/undiscord#635] [1. Questions & Answers]
Hey! First of all thanks for this very handy tool. I'm a bit of a noob when it comes to this so I was just wondering if using this tool to delete all my messages from a discord server could put the server itself and / or other members at risk? I know it's a potential risk for my own account but I was wondering about the serv' and the other members. 
Thanks!

**Comment by Clyde6790pGIT** [comment]
It should only delete your messages and bot messages related to it (ONLY IF YOU ARE AN ADMIN OR OWNER), unless you somehow set the author/userid to someone else's that will cause it to delete their messages (ONCE AGAIN ONLY IF U ARE AN ADMIN OR OWNER), so basically the risk is low.

**Comment by DanKawazko** [comment]
Not an admin nor owner. I'm a simple member so the risk should only be for me right? Cheers for the answer!
